docs/tools/cli/service/acl.rst

```diff
@@ -29,7 +29,7 @@ Adds an Aiven for Apache Kafka ACL entry.
   * - ``--username``
     - The username pattern: accepts ``*`` and ``?`` as wildcard characters
 
-**Example:** Add an ACLs for users with username ending with ``userA`` to ``readwrite`` on topics having name starting with ``topic2020`` in the service``kafka-doc``.
+**Example:** Add an ACLs for users with username ending with ``userA`` to ``readwrite`` on topics having name starting with ``topic2020`` in the service ``kafka-doc``.
```

Comment:
Hmm, I wonder if this commit was not intended to this PR. But since it is just a space, I'll merge together.